test/test618.js

```diff
@@ -0,0 +1,33 @@
+if (typeof exports === 'object') {
+    var assert = require("assert");
+    var alasql = require('..');
+}
+
+var test = '618'; // insert test file number
+
+describe('Test ' + test + ' - Where RTRIM will remove leading whitespace characters in an expression.', function() {
```

Comment:
Good catch! apparently both tabs and newlines should be ignored. 
Using `/^\s+/` actually includes tabs and whites space characters. So I switched to use `^[ ]+/` to very explicit. 

Also added the unit test as recommended.